LICENSE.txt

```diff
@@ -1,5 +1,5 @@
 Copyright (c) 2011 Lukas Lalinsky
-Copyright (c) 2011, 2012 MetaBrainz Foundation Inc.
```

Comment:
Yeah, that sounds like the safest & easiest thing to do, to just append 2015 (and it's advisable whenever major changes are made, but nobody ever remembers :)).